app/views/charge.njk

```diff
@@ -6,8 +6,13 @@
   {{ __("cardDetails.title") }}
 {% endblock %}
 
+{% set bodyClasses = "google-pay-unavailable" %}
 {% set mainClasses = "charge-new" %}
 
+{% block bodyStart %}
+  <script>document.body.className = document.body.className + ((window.ApplePaySession && window.ApplePaySession.canMakePayments()) ? ' apple-pay-available' : ' apple-pay-unavailable');</script>
```

Comment:
Any reason not to use `classList`? We’re already relying on it for reading classes from elements

Reply:
Yeah good point

Reply:
Updated PR